doc/source/serve/advanced-guides/external-scaling-webhook.md

```diff
@@ -0,0 +1,153 @@
+(serve-external-scale-webhook)=
+
+:::{warning}
+This API is in alpha and may change before becoming stable.
+:::
+
+# External Scaling Webhook
+
+Ray Serve exposes a REST API endpoint that you can use to dynamically scale your deployments from outside the Ray cluster. This endpoint gives you flexibility to implement custom scaling logic based on any metrics or signals you choose, such as external monitoring systems, business metrics, or predictive models.
+
+## Overview
+
+The external scaling webhook provides programmatic control over the number of replicas for any deployment in your Ray Serve application. Unlike Ray Serve's built-in autoscaling, which scales based on queue depth and ongoing requests, this webhook allows you to scale based on any external criteria you define.
+
+## Prerequisites
+
+Before you can use the external scaling webhook, you must enable it in your Ray Serve application configuration:
+
+### Enable external scaler
+
+Set `external_scaler_enabled: true` in your application configuration:
+
+```yaml
+applications:
+  - name: my-app
+    import_path: my_module:app
+    external_scaler_enabled: true
+    deployments:
+      - name: my-deployment
+        num_replicas: 1
+```
+
+:::{warning}
+External scaling and built-in autoscaling are mutually exclusive. You can't use both for the same application.
+
+- If you set `external_scaler_enabled: true`, you **must not** configure `autoscaling_config` on any deployment in that application.
+- If you configure `autoscaling_config` on any deployment, you **must not** set `external_scaler_enabled: true` for the application.
```

Comment:
The two points for negation of the other. Choose one of the points and remove the other?